certified-connectors/personr/personr.swagger.json

```diff
@@ -0,0 +1,491 @@
+                    }
+                  },
+                  "description": "response"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api-document-upload": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Upload Documents to an Applicant",
+        "operationId": "Api-document-upload",
+        "description": "Use this action to upload documents to an applicant, instead of sending them a link. You can only send one document at a time, when all required documents are uploaded the applicant will move to the pending status.  If you're uploading double-side documents, make sure to send FRONT_SIDE and BACK_SIDE in the idDocSubType parameter. When both sides are present and valid the verification step will be considered complete. Images must be uploaded in base64 format.",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "applicantId": {
+                  "type": "string",
+                  "description": "applicantId",
+                  "title": "Applicant ID"
+                },
+                "docType": {
+                  "type": "string",
+                  "description": "docType",
+                  "title": "Document Type (eg. ID_CARD)"
+                },
+                "docSubType": {
+                  "type": "string",
+                  "description": "docSubType",
+                  "title": "Document Subtype (eg. FRONT_SIDE)"
+                },
+                "docCountryISO": {
+                  "type": "string",
+                  "description": "docCountryISO",
+                  "title": "Document Country ISO (eg. AUS)"
+                },
+                "docFile": {
+                  "type": "object",
+                  "properties": {
+                    "filename": {
+                      "type": "string",
+                      "description": "filename",
+                      "title": "File Name + Extension (eg. document.jpeg)"
+                    },
+                    "contents": {
+                      "type": "string",
+                      "description": "contents",
+                      "title": "File Contents (Base64 string)"
+                    }
+                  },
+                  "description": "docFile"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/api-applicant-status": {
+      "post": {
+        "summary": "Retrieve Applicant Status",
+        "description": "Use this action to retrieve an applicant verification status. If the verification has been already reviewed, the response will contain data with more details about the result.",
```

Comment:
please keep all operations descriptions short to fit in the text box. please see our [docs](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).